kortex_driver/src/hardware_interface.cpp

```diff
@@ -1040,6 +1006,125 @@ void KortexMultiInterfaceHardware::sendTwistCommand()
   base_.SendTwistCommand(k_api_twist_command_);
 }
 
+void KortexMultiInterfaceHardware::resetFaults()
+{
+  if (in_fault_ == 0.0)
+  {
+    reset_fault_async_success_ = 1.0;
+    reset_fault_cmd_ = NO_CMD;  // clear the reset_fault_cmd
+    RCLCPP_INFO(LOGGER, "Arm is not in a faulted state. Returning SUCCESS from resetFaults().");
+    return;
+  }
```

Comment:
Do we expected to be in this state? The previous code didn't check for this, and restultFaults only gets called if `!std::isnan(reset_fault_cmd_)`.

Depending on how often we get here, the log output may be spammed.

Reply:
If the service is called when there is not a fault I didn't want the arm to be e-stopped so I added this guard.